src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -348,10 +348,21 @@ export default class ExportImportService {
         );
     }
 
-    private async createToggles(dto: ImportTogglesSchema, user: User) {
+    private async createOrUpdateToggles(dto: ImportTogglesSchema, user: User) {
```

Comment:
we had a bug in the previous import that you couldn't update toggles themselves. So e.g. if you changes toggle type in import it was ignored.

Reply:
Now we detect if feature exists and update it if needed. I wanted to put this code into the feature toggle service itself bu we need createOrUpdate within a project not a global createOrUpdate in any project so for now I decided to leave createOrUpdate in the import subdomain.